util/cops/deprecations.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rubygems
+      # This cop enforces that no outdated deprecations are present on RubyGems
+      # code base.
+      #
+      # @example
+      #
+      #   As of March, 2019
+      #
+      #   # bad
+      #   deprecate :safdfa, nil, 2018, 12
+      #   deprecate :safdfa, nil, 2019, 02
+      #
+      #   # good
+      #   deprecate :safdfa, nil, 2019, 03
```

Comment:
Just noticed that this example doesn't match the code. If we try to release a major version, and rubocop finds a method that's supposed to be deprecated in that same month, it will print an offense, although it's documented as "good" here. I'll fix it now.